apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -0,0 +1,297 @@
+
+/**
+ * Expects that this [Path] has a [parent][Path.getParent], that the parent holds all assertions the
+ * given [assertionCreator] creates for it and returns an [Expect] for the current subject of the assertion.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.parent(assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
+    ExpectImpl.path.parent(this).addToInitial(assertionCreator)
+
+/**
+ * Expects that [other] resolves against this [Path] and creates an [Expect] for the resolved [Path]
+ * so that further fluent calls are assertions about it.
+ *
+ * @return The newly created [Expect] for the extracted feature.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.resolve(other: String): Expect<Path> =
+    ExpectImpl.path.resolve(this, other).getExpectOfFeature()
+
+/**
+ * Expects that [PathWithCreator.path] resolves against this [Path], that the resolved [Path] holds all assertions the
+ * given [PathWithCreator.assertionCreator] creates for it and
+ * returns an [Expect] for the current subject of the assertion.
+ *
+ *  Use the function `path(String) { ... }` to create a [PathWithCreator].
+ *
+ * @return The newly created [Expect] for the extracted feature.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.resolve(path: PathWithCreator<Path>): Expect<T> =
+    ExpectImpl.path.resolve(this, path.path).addToInitial(path.assertionCreator)
+
+/**
+ * Helper function to create a [PathWithCreator] based on the given [path] and [assertionCreator].
+ */
+fun <E> path(path: String, assertionCreator: Expect<E>.() -> Unit) = PathWithCreator(path, assertionCreator)
+
+/**
+ * Expects that the subject of the assertion (a [Path]) is readable;
+ * meaning that there is a file system entry at the location the [Path] points to and
+ * that the current thread has the permission to read from it.
+ *
+ * This matcher _resolves_ symbolic links.
+ * Therefore, if a symbolic link exists at the location the subject points to,
+ * search will continue at the location the link points at.
+ *
+ * This assertion is not atomic with respect to concurrent file system operations on the paths the assertions works on.
+ * Its result, in particular its extended explanations, may be wrong if such concurrent file system operations
+ * take place.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") readable: readable): Expect<T> =
```

Comment:
Another word on ReplaceWith in the jdk8 extension of the infix API. You need to use the full qualified name for the pseudo keywords. For instance for this function you have to add:
```
ReplaceWith("ch.tutteli.atrium.api.infix.en_GB.toBe(ch.tutteli.atrium.api.infix.en_GB.readable)")
```
and you need to deprecated the pseudo-keywords as well pointing to the new pseudo-keyword (adjust name in case it changed)